gril/grilreply.c

```diff
@@ -894,8 +901,13 @@ GSList *g_ril_reply_parse_get_calls(GRil *gril, const struct ril_msg *message)
 		}
 
 		parcel_r_int32(&rilp); /* namePresentation */
-		parcel_r_int32(&rilp); /* uusInfo */
-
+		has_uusinfo = parcel_r_int32(&rilp); /* uusInfo */
+		if (has_uusinfo == 1) {
+			parcel_r_int32(&rilp); /* Type */
+			parcel_r_int32(&rilp); /* Dcs */
+			//Read byte array with user data and ignore it		
```

Comment:
Remove trailing whitespace.

```suggestion
			//Read byte array with user data and ignore it
```